README.md

```diff
@@ -32,6 +32,38 @@ module.exports = {
 }
 ```
 
+## Options
+
+### apiKey (required)
+
+An ApostropheCMS API key with permissions to make GET requests. [Read more about setting up API keys](https://a3.docs.apos.dev/reference/api/authentication.html#api-keys) in the Apostrophe documentation.
+
+### baseUrl (required)
+
+The base URL for API requests. Usually this will be the root domain of the Apostrophe website or application with no trailing slash. In local development it will be `http://localhost:3000` by default.
+
+### pieceTypes
+
+An array of Apostrophe piece type names. These will the same as the key name entered in the `app.js` `modules` object in the Apostrophe app. You may include core piece types that are not included by default, e.g, `apostrophe/file`, `apostrophe/image-tag`.
+
+### renderPages
+
+Defaults to `true`. Set this to `false` to avoid an additional request for each Apostrophe page in order to add the page's rendered HTML to the `AposCorePage` nodes.
+
+## Notes
+
+### Naming
+
+Apostrophe content node types in Gatsby's GraphQL API are all prefixed with `Apos`. So if you include your `article` piece type, it will appear in the GraphQL API as `AposArticle` nodes and the query collections will be `aposArticle` and `allAposArticle`.
+
+Apostrophe core piece types and pages are prefixed `AposCore`. For example, including the core `@apostrophecms/file` piece type will create `AposCoreFile` nodes.
+
+### Rendered content
+
+Apostrophe pieces are well suited to being delivered as structured, JSON-like data. Apostrophe pages, on the other hand, usually consist primarily of the flexible content "areas," populated with any number of different widget types. It powers Apostrophe's great in-context editing experience, but the raw data is not easy to use in a page template on your own.
+
+Unless you add `renderPages: false` to this source plugin's options, pages will appear with a `_rendered` property in Gatsby GraphQL queries. This property's value is a string of HTML, rendered using the relevant page template in the Apostrophe app. That HTML can be used in your Gatsby site to [programmatically create pages](https://www.gatsbyjs.com/docs/tutorial/part-seven/) using the right layout component and slug structure for your site.
+
 ## Planned features
 
 - Apos "areas" are rendered as HTML by default, rather than output as less usable raw data
```

Comment:
... Oh I see, this is still to come (separate PR?)

Reply:
Yep, it's a PR in core now. I'll then make a PR here, adding the query parameter.